src/deployment/default/index.tsx

```diff
@@ -18,8 +19,34 @@ const stubConsentContext = createContext<CookieConsent | undefined>(
 
 const defaultDeploymentFactory: DeploymentConfigFactory = () => ({
   chakraTheme: theme,
-  // This isn't ideal as it's the branded version. You can just remove the field to remove the welcome dialog.
-  welcomeVideoYouTubeId: "mREwMW69qKc",
+  appNameFull: "ml-trainer",
+  appNameShort: "ml-trainer",
+  AppLogo: (props: BoxProps) => {
```

Comment:
Will discuss what placeholder is suitable with Jonny but no need to block on that.
Will align with repo name change soon.